libsrc/leddevice/schemas/schema-apa102_ftdi.json

```diff
@@ -0,0 +1,27 @@
+{
+	"type": "object",
+	"required": true,
+	"properties": {
+		"output": {
+			"type": "string",
+			"title":"edt_dev_spec_outputPath_title",
+			"propertyOrder": 1
+		},
+		"rate": {
+			"type": "integer",
+			"title": "edt_dev_spec_baudrate_title",
+			"default": 5000000,
```

Comment:
Any benefit to add 

`"step": 100000,`

or go to a rateList

Reply:
it's for fine pam adjustment, I don't think it makes sense to limit user